ngraph/core/include/openvino/core/node.hpp

```diff
@@ -100,6 +104,13 @@ std::string node_validation_failure_loc_string(const Node* node);
     case element::Type_t::a: \
         rc = evaluate<element::Type_t::a>
 
+#ifdef NGRAPH_UNIT_TEST_ENABLE
```

Comment:
The same comment here.
We shouldn't have test artifacts in our public headers.

Reply:
Decided to keep ov::FuncionAccessor without macro until further better solutions.